src/pages/Home.js

```diff
@@ -0,0 +1,131 @@
+import { Link } from 'react-router-dom'
+import {Navigation} from '../components/Navigation'
+import '../components/clhome.css'
+import '../components/navigation.css'
+import biryaniimage from '../assets/images/biryaniimg.jpeg'
+import thaaliimage from '../assets/images/thalis.jpeg'
+import cakesimage from '../assets/images/cakes.jpeg'
+import chineseimage from '../assets/images/chinese.webp'
+import imageavtaar from '../assets/images/avatar-the-grinch-green-plant-recycling-symbol-food-transparent-png-1652162.png'
+import React,{useState} from "react"
+
+export default function Home(){
+
+    const [displayLogIn,setDisplayLogIn] = useState("none")
+    const [displaySignIn,setDisplaySignIn] = useState("none")
+    function clickLogInHandler(){
+        if(displayLogIn === "none")
+        setDisplayLogIn("block")
+        if(displayLogIn==="block")
+        setDisplayLogIn("none")
+    }
+    function clickSignInHandler(){
+        if(displaySignIn==="none")
+        setDisplaySignIn("block")
+        if(displaySignIn==="block")
+        setDisplaySignIn("none")
+    }
+    return(
+        <div className="container">
+            <div className="cards">
+
+<div className="card">
+
+    <img src={biryaniimage} alt="biryani" style={{width:"100%"}}/>
+    <h1>Biryanis</h1>
+    <p className="price">₹100* Onwards</p>
+    <p>Order from best resturants</p>
+    <p><Link to="/ProductListing">more</Link></p>
+  </div>
+  <div className="card">
+                <img src={thaaliimage} alt="biryani" style={{width:"100%"}} />
+                <h1>Thalis</h1>
+                <p className="price">₹100* Ownwards</p>
+                <p>Order from best resturants</p>
+                <p>< Link to="/ProductListing">more</Link></p>
+              </div>
+              <div className="card">
+                <img src={cakesimage} alt="biryani" style={{width:"100%"}} />
+                <h1>Cakes and Pastries</h1>
+                <p className="price">₹100* Onwards</p>
+                <p>Order from best resturants</p>
+                <p>< Link to="/ProductListing">more</Link></p >
+              </div>
+              <div className="card">
+                <img src={chineseimage} alt="biryani" style={{width:"100%"}}/>
+                <h1>Chinese</h1>
+                <p className="price">₹100* Ownwards</p>
+                <p>Order from best resturants</p>
+                <p>< Link to="/ProductListing">more</Link></p>
+              </div>
+  </div>
+    <div className="button-container">
+        <button className="login-btn-ecom" onClick={()=>clickLogInHandler()}>Login</button>
+        <button className="signup-btn-ecom" onClick={()=>clickSignInHandler()}>Signup</button>
```

Comment:
if you are not passing argumnets then just onClick={clickSignInHandle} is enough

Reply:
i tried but i am facing issue , would it be great if you could help me out here.Its not calling the method on click while doing onClick={clickSignInHandle}.